Whats-new-in-Swift-4.playground/Pages/Encoding and decoding.xcplaygroundpage/Contents.swift

```diff
@@ -32,10 +32,6 @@ let hand = [Card(suit: .clubs, rank: .ace), Card(suit: .hearts, rank: .queen)]
  Once you have a `Codable` value, you need to pass it to an encoder in order to archive it.
  
  Youʼll be able to write your own encoders and decoders that make use of the `Codable` infrastructure, but Swift will also come with a built-in set of encoders and decoders for JSON (`JSONEncoder` and `JSONDecoder`) and property lists (`PropertyListEncoder` and `PropertyListDecoder`). These are defined in [SE-0167][SE-0167]. `NSKeyedArchiver` will also support all `Codable` types.
- 
- SE-0167 is not fully implemented yet, so itʼs no surprise that the encoders crash in the toolchain I tested.
-
- [SE-0167]: https://github.com/apple/swift-evolution/blob/master/proposals/0167-swift-encoders.md "Swift Evolution Proposal SE-0167: Swift Encoders"
```

Comment:
We have to keep the reference link (line 37 and 38). I t's used in the sentence before.